.github/workflows/release.yml

```diff
@@ -25,6 +25,21 @@ jobs:
           NPM_EMAIL: ${{secrets.NPM_EMAIL}}
           NPM_USERNAME: ${{secrets.NPM_USERNAME}}
         run: yarn release
+  deploy:
+    needs: release
+    if: "contains(github.event.head_commit.message, 'chore(release)')"
```

Comment:
Not sure if it makes a difference, but at another place in the code we do this check with "startsWith(...)" instead of "contains()". I'd probably use that here too, just for consistency's sake.